apps/namadillo/src/registry/namadaAsset.ts

```diff
@@ -1,21 +1,33 @@
 import { Asset } from "@chain-registry/types";
+import { localnetConfigAtom } from "atoms/integrations/atoms";
+import { getDefaultStore } from "jotai";
 import namadaShieldedSvg from "./assets/namada-shielded.svg";
 
-export const namadaAsset = {
-  name: "Namada",
-  base: "unam",
-  address: "tnam1qy440ynh9fwrx8aewjvvmu38zxqgukgc259fzp6h",
-  display: "nam",
-  symbol: "NAM",
-  denom_units: [
-    {
-      denom: "unam",
-      exponent: 0,
-    },
-    {
-      denom: "nam",
-      exponent: 6,
-    },
-  ],
-  logo_URIs: { svg: namadaShieldedSvg },
-} satisfies Asset;
+const ADDRESS = "tnam1qxp7u2vmlgcrpn9j0ml7hrtr79g2w2fdesteh7tu";
+
+// We can't return "satisfies Asset" from fn
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+export const namadaAsset = () => {
```

Comment:
I'm removing this implementation and moving it to `namada-chain-registry` https://github.com/anoma/namada-interface/pull/1308

Do you think it could have some impact here?